executor/runtime/docker/networking.go

```diff
@@ -27,13 +27,14 @@ import (
 
 const (
 	defaultNetworkBandwidthBps = 128 * MB
-	vpctoolTimeout             = 45 * time.Second
+	assignVpctoolTimeout       = 120 * time.Second
```

Comment:
Given the tools are also in titus-executor, would it be straightforward to export the values and reuse them here and in the tools? It'd at least make the sharing more explicit.

If that's a pain then worst case could add comments here pointing to the other files.

Reply:
Done.